cmake/ActsExternSources.cmake

```diff
@@ -0,0 +1,52 @@
+set( ACTS_ACTSVG_SOURCE
+   "URL;https://github.com/acts-project/actsvg/archive/refs/tags/v0.4.40.tar.gz;URL_MD5;c8f3a7ac47db7f39cd4f215a5bbf148d" CACHE STRING "Source to take ACTSVG from")
+mark_as_advanced( ACTS_ACTSVG_SOURCE )
+
+set( ACTS_VECMEM_SOURCE
+   "URL;https://github.com/acts-project/vecmem/archive/refs/tags/v1.4.0.tar.gz;URL_MD5;af5434e34ca9c084678c2c043441f174" CACHE STRING "Source to take VECMEM from")
+mark_as_advanced( ACTS_VECMEM_SOURCE )
+
+set( ACTS_ALGEBRAPLUGINS_SOURCE
+   "URL;https://github.com/acts-project/algebra-plugins/archive/refs/tags/v0.22.0.tar.gz;URL_MD5;42bcaad8d19a2c773993a974816dfdf5" CACHE STRING "Source to take ALGEBRAPLUGINS from")
+mark_as_advanced( ACTS_ALGEBRAPLUGINS_SOURCE )
+
+set( ACTS_COVFIE_SOURCE
+   "URL;https://github.com/acts-project/covfie/archive/refs/tags/v0.10.0.tar.gz;URL_MD5;af59c6e2a1eebfa765b29f0af9fc70f7" CACHE STRING "Source to take COVFIE from")
+mark_as_advanced( ACTS_COVFIE_SOURCE )
+
+set( ACTS_DETRAY_SOURCE
+   "URL;https://github.com/acts-project/detray/archive/refs/tags/v0.65.1.tar.gz;URL_MD5;fbf57a881565fa6019d79d13409b588f" CACHE STRING "Source to take DETRAY from")
+mark_as_advanced( ACTS_DETRAY_SOURCE )
+
+set( ACTS_TRACCC_SOURCE
+   "URL;https://github.com/acts-project/traccc/archive/refs/tags/v0.10.0.tar.gz;URL_MD5;131399d26e3280c4d7f7ca2995efd256" CACHE STRING "Source to take TRACCC from")
+mark_as_advanced( ACTS_TRACCC_SOURCE )
+
+set( ACTS_DFELIBS_SOURCE
+   "URL;https://github.com/acts-project/dfelibs/archive/refs/tags/v20211029.tar.gz;URL_MD5;87fb09c5a11b98250f5e266e9cd501ea" CACHE STRING "Source to take dfelibs from")
+mark_as_advanced( ACTS_DFELIBS_SOURCE )
+
+set( ACTS_FRNN_SOURCE
+   "GIT_REPOSITORY;https://github.com/lxxue/FRNN;GIT_TAG;3e370d8d9073d4e130363faf87d2370598b5fbf2" CACHE STRING "Source to take FRNN from")
+mark_as_advanced( ACTS_FRNN_SOURCE )
+
+set( ACTS_GEOMODEL_SOURCE
+   "GIT_REPOSITORY;https://gitlab.cern.ch/GeoModelDev/GeoModel;GIT_TAG;4.6.0;PATCH_COMMAND;git am ${CMAKE_CURRENT_SOURCE_DIR}/0001-Add-option-to-skip-setting-up-json-completely.patch" CACHE STRING "Source to take GeoModel from")
+mark_as_advanced( ACTS_GEOMODEL_SOURCE )
```

Comment:
I fear this broke the patch apply from @paulgessinger 

cc @fredevb @asalzburger @krasznaa